### ISSUE TITLE
🖌️ Minor visual clarity upgrades

### ISSUE THREAD

**Issue by JFWooten4** [WhyDRS/Database#17]
[Excited](https://github.com/WhyDRS/Database/commit/55917b81125f17ebcc6be6b95a6cfd1dd393868b#commitcomment-146866960) to see the recent progress in stylizing the database for easy use. 📈 Thank you for all the incredible contributions!

---

Might we clean up a couple small display items? 🖥️

### Sidebar Selected Font

Might we make this text [color](https://github.com/WhyDRS/DAO-docs/blob/main/static/imgs/brand/colors.md) darker (for contrast)? 

<img width="227" alt="Screenshot 2024-09-18 at 5 16 28 AM" src="https://github.com/user-attachments/assets/aa97af1d-26d9-4d0b-9f60-c314d9e8d82c">

### Header Title Spacing

Maybe this is just me, but do you think `IR Email` looks better than `I R Email`? 📝

<img width="280" alt="Screenshot 2024-09-18 at 5 27 30 AM" src="https://github.com/user-attachments/assets/134195aa-a0d8-4e31-94f1-02fdab502f5a">

**Comment by JamesAlfonse** [comment]
Agreed on both thoughts here. I also wanted to reverse the colors on the selected/unselected options so that the unselected options would be grayed out instead. The column names don't have any spaces in them, which is where I think the issue stems from. I had run into some problems when they had spaces in them, it might be good to circle back and figure out a solution for this.
Also noticed that enabling and disabling columns on the site doesn't shrink the table; enabling all the columns then disabling all but one will leave a massive amount of space for just that one column.

**Comment by JamesAlfonse** [comment]
![Screenshot 2024-10-20 at 5 31 06 PM](https://github.com/user-attachments/assets/35d17e3e-7c88-42dd-8d81-753d2bc399f6)

Added readability to these buttons, but have not been able to make progress on adjusting the colors between the active and non-active buttons. The active buttons look like they're pressed in, but adjusting the colors in the css file for the section below does not work. #134F9B should be a blue color. It's possible that there's a conflict between the css code and the datatables library causing this. One solution could be to download the full library and include it in this repository to have more visible control over the styling. Regardless, I would prefer to look at other stack alternatives first before proceeding with this option #8. I will look into fixing the headers next.

/* Styles for active buttons using aria-pressed attribute */
div.dt-button-collection .dt-button[aria-pressed="true"] {
    /* Set background color for active buttons */
    background-color: #134F9B !important;
    /* Set text color for active buttons */
    color: #FFFFFF !important;
}

**Comment by JFWooten4** [comment]
@JamesAlfonse Far above my paygrade. 🤯 Thank you for both tackling this crucial challenge, and for sharing the process with us here&mdash;both via your thoughts and documenting actions which can [so often](https://x.com/xximpod/status/1853995897840963882) get lost in git commit gibberish.[^as] Seriously appreciate the choice to analyze the framework approach before hastling with the unresponsive colors. 🎨

[^as]: And thanks for un-assigning yourself! I know this is only available to people who have `write` access, but it's super useful when I'm scanning through issues. In my view, the worst feeling you can have is when you're working on something (as shown through either an assignment declaration or a draft PR mentioned in comments on the issue), but then someone else just drops the same work already completed in a new PR!

**Comment by JamesAlfonse** [comment]
Issue #39 covers fixing the buttons for enabled columns. Closing this issue for now.
